index.html

```diff
@@ -1,9 +1,178 @@
+          CSS. I've been hooked ever since.
+        </p>
+
+        <div class="figure_container">
+        <figure class="about_figure">
+          <img
+            src="./images/youngSara.jpg"
+            alt="sara smiling amongst many young cellists"
+          />
+          <br />
+          <figcaption>
+            I began my career as a cellist at a young age.
+          </figcaption>
+        </figure>
+        <figure class="about_figure">
+          <img
+            src="./images/mobile_sara.jpg"
+            alt="sara playing cello with eyes closed"
+          />
+          <br />
+          <figcaption>I went to music school and honed my craft.</figcaption>
+        </figure>
+        <figure class="about_figure">
+          <img
+            src="images/coding_screenshot.avif"
+            alt="html coding text"
+          />
+          <br />
+          <figcaption>
+            I developed a passion for coding after an injury.
+          </figcaption>
+        </figure>
+        <figure class="about_figure">
+          <img
+            src="./images/coding_frenzy.jpg"
+            alt="Hands typing on keyboard overlaid with coding text"
+          />
+          <br />
+          <figcaption>I am now an aspiring developer!</figcaption>
+        </figure>
+        </div>
+      </div>
+
+      <br />
+      <hr id="projects_hr"/>
+
+      <div class="projects_container">
+        <h2 id="projects_heading">Projects</h2>
+        <section class="projects_portfolio">
+          <a class="projects_a" href="https://github.com/sgcook/portfolio" target="_blank">Portfolio</a>
+          <img id="img_portfolio" class="projects_img" src="./images/portfolio.png" alt="horizontal portfolio of images" />
+          <p class="projects_p">Practice using HTML and CSS</p>
+        </section>
+
+        <section class="projects_highlands">
+          <a class="projects_a" href="https://github.com/sgcook/Highlands" target="_blank">Highlands Page</a>
+          <img id="img_highlands" class="projects_img" src="./images/highlands.png" alt="screenshot of sara's highland website" />
+          <p class="projects_p" >Practice using Flexbox</p>
+        </section>
+
+        <section class="projects_cello">
+          <a class="projects_a" href="https://github.com/sgcook/website" target="_blank">Cello Professional Site</a>
+          <img id="img_cello" class="projects_img" src="./images/cello.png" alt="screenshot of sara's cello website" /></td>
+          <p class="projects_p">My first coding project built with CSS and HTML</p>
+        </section>
+      </div>
+
+      <br />
+      <hr id="contact_hr"/>
+
+
+      <div class="contact_container">
+        <h2 id="contact_heading">Contact</h2>
+        <form class="form"> 
+          <fieldset class="contact_details">
+            <legend>Personal Details</legend>
+            <label id="name">
+            Name:
+            <input type="text" name="name" />
+            </label>
```

Comment:
You have your input field inside your label here, these should be sibling components and not have a child parent relationship. This is the same for the other form elements